Readme.md

```diff
@@ -37,6 +37,26 @@ items = [1,2,3]
 Parallel.each( -> { items.pop || Parallel::Stop }) { |number| ... }
 ```
 
+Reduce will return results colleced from each worker. So you need to merge them on the end.
```

Comment:
Maybe write a bit about what you just wrote in the comment ...

```
To avoid overhead from reducing big result sets at the end, use `.reduce` to bla bla
```

Reply:
So I added comment similar to that